Scripts/script-GetDuplicatesMl.yml

```diff
@@ -0,0 +1,551 @@
x97M\x12\xa8\xf6\xa8\x93\xc3\x87\x11\tT\xfa\xd5wC\x9b\xafL\xa0\x1a;\xbc\x1e\xf4\xd3\x8aDj\xc8\xe4\xf5S\xb3>$Q\x13\x7fmr\xf3\xe7\x83g\xa9\x1eG\xbbMns\xe9,\xf5\xd7\xacI\':7I\xa6\xc6O\xd4\xf9m\xa9\x94\xe9\x85\x0f\x9a\x17\x0e\xbcD\xddX\xe4ts\xce\xba\xcb\x94\xff^\x9b\xfa[\x7f\xbaB\xe5\xa8\xfb\x0cu,\xbeJYJ\xc6z\x15:\xa7\xc8\xf4:\xdbS\xa8>\xa3-\x03\x9f\xfb\xdc\xa0$\xfb\xbf\\\xfb\xbc\xe5\x06\xb5\xbf\xc7A\xb3e^\xa9\xd4\x9c\xd6g"\xbf\xba\xdd\xa6F=x\xd2E\x8b\xb9M\xd5\x92:\xac\xb98\x8d\xba\\k\xffR\xcd\xc1\xe9T\xf0\xc2%yk\xf4\xeeQ\xfb\xd2<\xa6d\x8f\xcb\x91\xf5\xc3\xd8/9\xd4\xa9\xedv\xa3/\xa6?\xa2\x0e\x0fJ\xce]\xab\xfe\x82Bn\xbd\xab\xaf\xd5}Ay\x87\xfe1\xf1\x97\x17/(f\xea\xedgk\xdb\xbd\xa42\xa6l96r\xea+j\xa44\xb0\xbc\xd9\xaf(\xed\x18{\xad\x84\xacW\xd4\xec\x84\x804\xba\xe0\x15\x95y\xef\xf5\xc4\x87\xd5_S\xc9\x16]\xebG\xac{M\xb9g\xbb\xefCi\xaf\xa9\xd4\x17\xeb\xe6\xc4g\xbe\xa6\x86\x05\xd6)\x1a\xfa\xec5\xb5\xd7\xc1b~\xc0\xf3\xd7T\xa7\xc6\xf1?l{\xf1\x9ar{\xf9dC\xb7\x97\xaf\xa9)QO\xf6F\xe8\xbe\xa1\x8a\xed\xad\x9b&\x86\xbe\xa1"\xda\xa6\xdb\x99\x85\xbd\xa1\x9a\x8d\xf6>\xfee\xe9\x1b\xea\xf21\xfd\x11_k\xbc\xa5\x86\x0c<\xd46\xbe\xd6[*\xeeV\x88\x9bs\x97\xb7\xd4\xceY\x19\xa9=5\xdeQ\xb7z6[\xba}\xd4;\xea\xc9\xb4\xcb\xd4\xb8F\xef\xa9\x07\x87\xe7\xcd\xb4t\xfd@]\xda\xe2t\xa4\xf5\xb6\x0fT\xc9\xfba\xbd\x9f}\xa2\xb6:\x1d\tA}\xd4\xe8\x03\xd6\x01-\x8b_\xaa\xd1\x1d\xbb\x9f\xbcqz\xa7:}\xba\xee\x06\xfb\xdf\xc3\xaa\xd1\xa2\x84]fO,5\xe8\xb6o\xd7>\xb0\xe9\xabAK\x8b\x9d\xb4\x08\xd6\xa0\xc7,=\x13j9G\x83>\xf8$\xe4pqSM\xbaS\x14\x9d\xe5qT\x93\xfex\xe9\x99\xb3\xc9\tM\xfaJ\xb7\x81\xd7\xa7fh\xd2\xe7\xd7\x1d7\xb8\xa3\xa5Eo\xb0\xdfz\x08\r\xd6\xa2w\x9cL0\xec<\\\x8b>\xb6\xb8z\xb1\xd1|-\xfa\xfe\x9d9!\x99W\xb4\xe8\x11E6\xc7kt\xa8N\xaf]V/l\x84\x94a\xc9\x16q\xe3wW\xa7\xa5\xfdr\xef\xeb\xcbZ\xb4g{\x91\xed\xe9\x80\xda\xf4^\xdf\x0c\xe6\xb4I=\xbaW\xd3\xc2\xf1_S\xea\xd1^\xb3\xd3\xc3\x9b\xd7\xd6\xa5\xcd\xeb\xd5zsMW\x97n\x15Z\xed\xee\xa6\x06\xba\xf4\x91W\x9fS{\x19\xe8\xd2\xb1\xe3g\xa7/\xb5i@w\xbc\xd2\xadn\x90]\x03\xfaq\xec\x9f\x89\xb3g4\xa0k.l\xbemP\\\x03z@\xc3E\x0b\'\x9b\x1a\xd0\xcfF;\x9e\xda\xf6\xd8\x88^?uC\x8e\xd7\xf0F\xb4^\xc7\xee\x9fo\x8emDw\x1fx]\xef\x8e\x94\xd7M\xef\x1d{5\xbd1=e\x90\xb1\x81dNc\xfa(57\xa7\xc1\x8e\xc6t\xb5\xf5\xda\xf7\xec?7\xa6GEv6\x18\xd4\xb5\t]<\xffB\xaa\xe1\xa4&\xf4\x94\xac\rC\xee\xcfnB\xdfP\xcb8\x93X\xcf\x98\xbe6\xe1\xd6\xc4$\x17c\xfa\xf2\xb3\x0b\x89\x07]\x8d\xe9\xfcE\xe6\xfe\xdb\x06\x18\xd3\xb5\xab\xb5\xed+\x19lL\x1f5\x9e_\xd8k\xa91}v\xa9\xdd\x9d\xc7[[\x94\x94\x7f\x91\xbe\xad\x05=6\xce\xf9r\xf6\xf9\x16t\xc06\xf5\x13\xce\x05-\xe8/>\xef4w\xa9\x9b\xd2\xa6~\xd3\xe2B\xeb\x98\xd2\x17\xdb\'\xbd\xb8/ez\x8f\xe9\x9d\xbb\xd75\xa5m-w\x1f4\xd15\xa5c\x06O\x0e\xeb\xdd\xcc\x94\x16\x1f|\xb2\xd2o\x83)}\xd9\xdd\xa5`\xcbFS\xda!"6\xab\xe6fS\xfa]\x9egB\xfaVS\xfac\xd7\xa0\xabz\xd1\xa6\xb4t{h~\xfd\x9d)\x1d]\xd0o\t\xf5\xde\x94>\x99\xab\x7f2A\xca\xc0\xe2\xf9\xce\xeb\xc5ft\xe2\x1b\xf3\xa6+\x83\xcc\xe8F9#\xd7\x0e\x98iF\xb7;\xf0\x9b\xcd\x81EftJ\xef\xb4\xc1\x9f\xee\x9a\xd1.\xa3"\x8d&5oE\xfb\xe7\x7f\xbcp\xa1\x8f\x94\x
```

Comment:
Consider the equity of the IP base on this subnet mask (a valid value is a number between 0 - 32). Default is 32 which means an exact match of the IP addresses.